src/neptune/internal/operation_processors/async_operation_processor.py

```diff
@@ -73,35 +72,25 @@ def __init__(
         self._container_id = container_id
         self._container_type = container_type
         self._backend = backend
+        self._lock = lock
+        self._sleep_time = sleep_time
         self._batch_size = batch_size
         self._last_version = 0
         self._consumed_version = 0
         self._consumer = self.ConsumerThread(self, sleep_time, batch_size)
-        self._drop_operations = False
 
         # Caller is responsible for taking this lock
         self._waiting_cond = threading.Condition(lock=lock)
 
-        if sys.version_info >= (3, 7):
-            try:
-                os.register_at_fork(after_in_child=self._handle_fork_in_child)
-            except AttributeError:
-                pass
-
     @staticmethod
     def _init_data_path(container_id: UniqueId, container_type: ContainerType):
         now = datetime.now()
         container_dir = f"{NEPTUNE_DATA_DIRECTORY}/{ASYNC_DIRECTORY}/{container_type.create_dir_name(container_id)}"
-        data_path = f"{container_dir}/exec-{now.timestamp()}-{now.strftime('%Y-%m-%d_%H.%M.%S.%f')}"
+        data_path = f"{container_dir}/exec-{os.getpid()}-{now.timestamp()}-{now.strftime('%Y-%m-%d_%H.%M.%S.%f')}"
```

Comment:
Let's move the process id to the end. Currently, it breaks "sorting" capabilities :D.

Reply:
Done.